### DISCUSSION TITLE
My current Recent Activity design

### DISCUSSION THREAD

**Discussion by Andre601** [Readme-Workflows/recent-activity#18] [Show and tell]
Just want to quickly share my Recent activity design I have:
![image](https://user-images.githubusercontent.com/11576465/117688386-0243dc80-b1b9-11eb-85ac-18ec35dfed86.png)

**Comment by abhijoshi2k** [comment]
That looks awesome!!!
I don't understand markdown very well can you please tell me what does `![issueOpened]` actually do?

**Reply by Andre601** [reply]
[google]: https://google.com

Those are reference links for images.

I essentially have `[text]: url` at the bottom of the file (hidden when rendered) that allow me to quickly put links at multiple places in the markdown file.

For example, making `[google]: https://google.com` in this comment itself allows me to just use `[google]` and it turns into [google].
It's also case-insensitive so [GoOGlE] works too and keeps the capitalisation.
Alternatively can I also use `[text][google]` to have a different text displayed, in this case [text][google].

And obviously by prefixing it with a `!` can it be treated like an image.

If that doesn't explain it, do I have a section in a dedicated page all about GitHub flavourited Markdown (Although this is basic markdown): https://www.andre601.ch/GitHub-Markdown/github/general/#reference

**Reply by abhijoshi2k** [reply]
I get it now. Thanks.
The info on your website is super cool. Loved the way you have explained and presented.

**Reply by Andre601** [reply]
Thanks. I enjoyed making the page. And since GitHub's started guide to markdown is fairly limited and doesn't even mention all features available do I share this one as the unofficial Guide for GHFM (GitHub Favourited Markdown)

**Reply by PuneetGopinath** [reply]
Instead of saying unofficial one you could say Andre601's guide to GHFM

**Comment by Andre601** [comment]
GitHub added new icons for closed (unmerged) and draft PRs
Not sure what to think of those.

![image](https://user-images.githubusercontent.com/11576465/118359483-fcb51080-b583-11eb-9b01-06784261f4e9.png)

Draft is this one:
https://octicons-primer.vercel.app/octicons/git-pull-request-draft-16

**Reply by abhijoshi2k** [reply]
Is there an event for draft pr?

**Reply by Andre601** [reply]
There is `payload.pull_request.draft` in the `PullRequestEvent` that returns a boolean.

But I can't tell if there's an event in the history that tells if a Draft was turned into a final PR

**Reply by abhijoshi2k** [reply]
If draft was turned into pr, I think it'll fire a pr event with `draft: false`

**Reply by PuneetGopinath** [reply]
I think the message can be something like `🥁 Drafted a pull request #1 in Readme-Workflows/recent-activity`

**Reply by PuneetGopinath** [reply]
> If draft was turned into pr, I think it'll fire a pr event with `draft: false`

Could be, let's check it out in recent-activity-template repo

**Reply by Andre601** [reply]
> I think the message can be something like `🥁 Drafted a pull request #1 in Readme-Workflows/recent-activity`

I think a pencil would be better.
`✏️ Drafted a Pull request #1 in Readme-Workflows/recent-activity`

**Reply by abhijoshi2k** [reply]
> > If draft was turned into pr, I think it'll fire a pr event with `draft: false`
> 
> Could be, let's check it out in recent-activity-template repo

I'm planning to start working on {DATE} tomorrow.
After that I'll work on adding other events.
Can you create a draft PR on recent-activity-template repo and check?

**Reply by PuneetGopinath** [reply]
> I'm planning to start working on {DATE} tomorrow.
> After that I'll work on adding other events.
> Can you create a draft PR on recent-activity-template repo and check?

Yes, will check now.
